### PR TITLE
Bugs/several bugs

### LINKED ISSUE #4: Molecule test with Docker (and therefore, Travis CI) doesn't work
Despite the tests work perfectly with Vagrant in a local environment, due to problems with Docker network interface is not possible to do it with Docker

### PR DESCRIPTION
**Pull request by jnogol**
Fixes #4 #10 #11

### CODE REVIEW

File: tasks/plugins.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@
     params:
       url: "{{ jenkins_url }}"
       url_username: "{{ jenkins_admin_username }}"
-      url_password: "{{ jenkins_admin_password }}"
+    url_password: "{{ jenkins_admin_password }}"
```
Comment (jmonterrubio) [inline tasks/plugins.yml]: Is this ok? I mean, the indentation :)

Comment (sorobon) [inline tasks/plugins.yml]: It may be an indentation error?

Comment (jnogol) [reply tasks/plugins.yml]: nope, actually this is the fix xD. But now you mentioned it, it doesn't look pretty clear this way. I'll put this line below the "state" one and add a comment then

Comment (jnogol) [reply tasks/plugins.yml]: @jmonterrubio

**Review by sorobon — APPROVED** [review]

**Review by jmonterrubio — APPROVED** [review]

**Pull request merged**